docs/index.md

```diff
@@ -4,6 +4,9 @@ title: "Mocha - the fun, simple, flexible JavaScript test framework"
 description: "Mocha is a feature-rich JavaScript test framework running on Node.js and in the browser, making asynchronous testing simple and fun."
 ---
 
+<!-- Disable MD051 due to the rule not aware of this file is specifically converted to HTML -->
```

Comment:
please describe MD051 or provide a link to its description in a comment in this file